res/res_pjsip_rfc3329.c

```diff
@@ -54,6 +75,15 @@ static void rfc3329_incoming_response(struct ast_sip_session *session, struct pj
 		goto out;
 	}
 
+	if (!datastore
+		&& (datastore = ast_sip_session_alloc_datastore(&rfc3329_store_datastore, "rfc3329_store"))
+		&& (store_data = ast_calloc(1, sizeof(struct rfc3329_store_data)))) {
```

Comment:
If the datastore can't be allocated, what will the resulting behavior be?

Reply:
I think it might be worthwhile to do a warning in this case, and communicate with the user what has happened and how it is still attempting.

Reply:
That makes a lot of sense. Done.